example/make_public.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# BSD 3-Clause License
+#
+# Copyright (c) 2020, Science and Technology Facilities Council.
```

Comment:
Oh my life, this has been a while! Apologies. Could update to 2020-24 now?

Reply:
Oh no, not your fault. That's a copy&paste artefact (and I've fixed that)